.github/tests/upstream-authority-cert-manager/cert_manager_ca.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
can we use the `kebab-style-filenaming` instead of `snake_case` like we do for other files as well.